projects/packages/videopress/src/client/admin/components/video-details-card/style.module.scss

```diff
@@ -28,6 +20,14 @@
 		border-radius: var( --jp-border-radius );
```

Comment:
I noticed that on Figma the mobile version of the thumbnail has no border radius.

Reply:
awesome catch! 🔎